helm/values.yaml

```diff
@@ -11,6 +11,34 @@ ingress:
   labels:
     ccs.ornl.gov/externalRoute: "true"
```

Comment:
Just noticed a hardcode ornl hostname here. @jasonb5

Reply:
@sashakames I'm going to address the default in another PR.